Library/Homebrew/extend/os/ENV/super.rb

```diff
@@ -0,0 +1,5 @@
+require "ENV/super"
```

Comment:
Fixing this revealed that this line itself should have been `require "extend/ENV/super"` which raises the question how logical the proposed directory structure is, or to be more specific, how logical it is that the platform-specific wrapper omits the `extend/` prefix. (I'm thinking here that `<path>` should always translate to `extend/os/<path>` and `extend/os/mac/<path>`.)